### PR TITLE
Add another integration, fix whitespace around punctuation

### PR DESCRIPTION
**Pull request by astro**
## Reason for this PR

Except in French, there is no whitespace preceding punctuation marks.

## Description of changes

Links to a project that makes it easy to run NixOS VMs on Firecracker.

## License acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the MIT-0 [1] license.

[1] https://github.com/aws/mit-0

## PR checklist

`[Author TODO: Meet these criteria & check the boxes.]`
`[Reviewer TODO: Verify checked boxes. Request changes if criteria not met]`

- [ ] The reason for this PR is clearly provided (issue no. or explanation).
- [ ] The description of changes is clear and encompassing.
- [x] All commits in this PR are signed (`git commit -s`).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -91,39 +91,28 @@
...
-            , containerd via
-            <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>
-            ,
-            <a href="https://fly.io">Fly.io</a>
-            ,
+            <a href="https://appfleet.com/">appfleet</a>,
+            containerd via
+            <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>,
+            <a href="https://fly.io">Fly.io</a>,
             <a
```
Comment (xmarcalx) [inline index.html:98]: not: we can align this line to the previous style too with everything in one line

`<a href="https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support">Kata Containers</a>`

File: index.html [hunk 2]
```diff
@@ -91,39 +91,28 @@
           </p>
           <p>
             Firecracker is used by/integrated with (in alphabetical order):
-            <a href="https://appfleet.com/">appfleet</a>
-            , containerd via
-            <a href="https://github.com/firecracker-microvm/firecracker-containerd">firecracker-containerd</a>
-            ,
-            <a href="https://fly.io">Fly.io</a>
-            ,
+            <a href="https://appfleet.com/">appfleet</a>,
```
Comment (xmarcalx) [inline index.html:94]: can we apply the same formatting rule for consistency to the code starting from line 270 please?

File: index.html [hunk 3]
```diff
@@ -117,8 +117,10 @@
             (via
             <a href="https://github.com/weaveworks/ignite">Weave Ignite</a>
             )
-            , and
+            ,
             <a href="https://webapp.io">webapp.io</a>
+            , and
+            <a href="https://github.com/astro/microvm.nix">microvm.nix</a>
```
Comment (xmarcalx) [inline index.html]: glad to see new users :) welcome to Firecracker community!

**Review by xmarcalx — CHANGES_REQUESTED** [review]
Hi @astro,

Thank you very much for the changes and welcome to Firecracker community.
I left some minor comments. Would you be ok for you to address them?

File: index.html (hunk 3 above)
Comment (astro) [reply index.html]: Thank you for your review. I resolved the issues.

Comment (xmarcalx) [reply index.html]: thanks for the follow up!

**Review by xmarcalx — APPROVED** [review]
Thanks for resolving the issues!

**Pull request merged**